### PR TITLE
Initial Discord Rich presence

### PR DESCRIPTION
**Pull request by io-june**
thanks @occune for initial work
put some edits there
doesnt work on macos(waiting on nuget update again)
btw future reference: https://discordapp.com/developers/docs/rich-presence/how-to

### CODE REVIEW

File: SS14.Client/SS14.Client.csproj [hunk 1]
```diff
@@ -75,6 +78,9 @@
     <PackageReference Include="SharpFont" Version="4.0.1" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="GodotSharp">
+      <HintPath>..\SS14.Client.Godot\.mono\assemblies\GodotSharp.dll</HintPath>
```
Comment (PJB3005) [inline SS14.Client/SS14.Client.csproj]: Can you undo this change? Your IDE messed up the project references.

Comment (io-june) [reply SS14.Client/SS14.Client.csproj]: fuck

Comment (io-june) [reply SS14.Client/SS14.Client.csproj]: fuck yeah*

**Review by PJB3005 — COMMENTED** [review]
Aren't you supposed to register some kind of periodic timer for the RPC client?

**Comment by io-june** [comment]
Nope, timer was needed to call invoke, but current version works without it now
